### DISCUSSION TITLE
Unable to create user

### DISCUSSION THREAD

**Discussion by tang-xiaoq** [apache/airflow#30695] [General]
When I execute ===“
>  airflow users create \
>     --username admin \
>     --firstname Peter \
>     --lastname Parker \
>     --role Admin \
>     --email spiderman@superhero.org

‘’

I cannot create a user：error====={manager.py:216} ERROR - Error adding new user to database. unsupported hash type sha256
Failed to create user

Please tell me where the problem，thank you。

**Comment by potiuk** [comment]
You have not shown the full stacktrace and you have not mentioned your Database, version, nor which Airflow version you are using, but likely you have an old DB client and new DB server  and your DB server refuses the client connection.

You need to search for similar issues for your database on how to fix the problem  - likely you need to enable older version of signture algorithm in your DB or upgrade the client library you are using, or maybe use older version of the DB server you have or (most likely) upgrade to latest version of Airflow and DB client libraries you have in your system.
